flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_ADIRS.js

```diff
@@ -1,28 +1,34 @@
-class ADIRS {
-    constructor(num) {
-        this.num = num;
-        // 0 -> OFF
-        // 1 -> NAV
-        // 2 -> ATT
-        this.knob = -1;
+class ADIRU {
+    constructor(number) {
+        this.number = number;
+
+        this.KNOB = Object.freeze({
+            OFF: 0,
+            NAV: 1,
+            ATT: 2
+        });
+        this.knob = this.KNOB.OFF;
 
         // -1 -> no timer
         //  0 -> timer finished
         // >0 -> timer remaining
         this.timer = -1;
-        this.onBatTimer = 0; //start at 0 to prevent light from coming on when starting with ADIRS aligned
+
+        // Start at 0 to prevent light from coming on when starting with ADIRS aligned
+        this.onBatTimer = 0;
     }
 
     init() {
-        this.knob = SimVar.GetSimVarValue(`L:A32NX_ADIRS_KNOB_${this.num}`, 'Enum');
-        SimVar.SetSimVarValue(`L:A32NX_ADIRS_TIMER_${this.num}`, 'Seconds', -1);
+        this.knob = SimVar.GetSimVarValue(`L:A32NX_ADIRS_KNOB_${this.number}`, 'Enum');
+
+        const startingAligned = this.knob === this.KNOB.NAV;
+        this.timer = startingAligned ? 0 : -1;
```

Comment:
This is the actual fix. Previously this would always start with `-1` and therefore the alignment timer would run again. I'm not sure why _sometimes_ the issue didn't occur. That still baffles me 😀.